app/components/audit_log/Index.js

```diff
@@ -87,17 +83,28 @@ class AuditLogList extends React.PureComponent<Props, State> {
       </Panel.Section>
     );
   }
+  renderFooter() {
+    if(this.props.organization.auditEvents) {
```

Comment:
Is this condition not already managed by the `ShowMoreFooter`?

Reply:
It was to get rid of this warning in console:

`Expected prop `connection` to be supplied to `ShowMoreFooter`, but got `undefined`. Pass an explicit `null` if this is intentional`

Since `this.props.organization.auditEvents` returns `undefined` if it's not been loaded yet by Relay...

Reply:
Oh! Could we key it off the `isMounted` variable so it’s more explicit, then?

Reply:
Cool, done.